packages/core/src/components/accordion/accordion.tsx

```diff
@@ -132,6 +132,21 @@ export class Accordion implements ComponentInterface, AccordionInterface {
     openSections.forEach((section) => this.controlOpenAttribute(section, false));
   }
 
+  // These checks are needed for a React timing issue.
```

Comment:
Tsja.. echt overtuigend klinkt dit natuurlijk niet. Ik stel voor dat we dit mergen zodat de afnemers verder kunnen maar dat we dit in een later issue oppakken.